css/selectors/-moz-submit-invalid.json

```diff
@@ -15,12 +15,40 @@
             "edge": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": "4"
-            },
-            "firefox_android": {
-              "version_added": "4"
-            },
+            "firefox": [
+              {
+                "version_added": "88",
+                "notes": "From Firefox 88 the feature has been placed behind a flag. See <a href='https://bugzil.la/1694129'>bug 1694129</a>.",
```

Comment:
Doing something little remarkable here, so leaving a note for posterity: ordinarily, I'd say to leave this note out (or to only have the `See bug` portion). But this is an odd case of something going behind a pref after being unpreffed, so I'll leave it as is.